pkg/apis/ocs/v1/storagecluster_types.go

```diff
@@ -104,11 +104,13 @@ type StorageClusterStatus struct {
 	// +optional
 	FailureDomain string `json:"failureDomain,omitempty"`
 
-	StorageClassesCreated       bool `json:"storageClassesCreated,omitempty"`
-	CephObjectStoresCreated     bool `json:"cephObjectStoresCreated,omitempty"`
-	CephBlockPoolsCreated       bool `json:"cephBlockPoolsCreated,omitempty"`
-	CephObjectStoreUsersCreated bool `json:"cephObjectStoreUsersCreated,omitempty"`
-	CephFilesystemsCreated      bool `json:"cephFilesystemsCreated,omitempty"`
+	StorageClassesCreated       bool                `json:"storageClassesCreated,omitempty"`
+	CephObjectStoresCreated     bool                `json:"cephObjectStoresCreated,omitempty"`
+	CephBlockPoolsCreated       bool                `json:"cephBlockPoolsCreated,omitempty"`
+	CephObjectStoreUsersCreated bool                `json:"cephObjectStoreUsersCreated,omitempty"`
+	CephFilesystemsCreated      bool                `json:"cephFilesystemsCreated,omitempty"`
+	ExternalSecretFound         bool                `json:"externalSecretFound,omitempty"`
+	ExternalStorageClasses      []ExternalResources `json:"externalStorageClasses,omitempty"`
```

Comment:
Move this above the booleans and provide a comment on what it's for. See `FailureDomain`.

Reply:
Are you talking about just the `ExternalStorageClasses` or both the `ExternalStorageClasses` and `ExternalSecretFound`?.  I also believe to move the latter with the former as both are only being used when there's an External Cluster.
Wdyt?.